src/com/epam/javacourse/homework4/Main.java

```diff
@@ -15,12 +18,12 @@ public static void main(String[] args) {
         GreenhouseManagement.TemperatureSensor sensor = new GreenhouseManagement.TemperatureSensor(25.0);
         double currentTemperature = sensor.getTemperature();
         System.out.println("Current temperature: " + currentTemperature + " degrees Celsius");
-        GreenhouseManagement myGreenhouse = new GreenhouseManagement();
-        Plant.useGardenTool(myGreenhouse, "Watering can");
+        Plant.useGardenTool(greenhouse, "Watering can");
 
         boolean exit = false;
         do {
-            System.out.println("Enter 1 to create a new plant, 2 to receive available plants, 3 to buy a new plant, 4 to display all plants in the basket, 5 to update a plant, 6 to remove a plant, 7 to water the plants, 8 to set the temperature, 9 to exit:");
+           // System.out.println("Enter 1 to create a new plant, 2 to receive available plants, 3 to buy a new plant, 4 to display all plants in the basket, 5 to update a plant, 6 to remove a plant, 7 to water the plants, 8 to set the temperature, 9 to exit:");
```

Comment:
Having comments not related to code documentation in production code is not a good idea.